### ISSUE TITLE
un-monitored does not work

### ISSUE THREAD

**Issue by Vansmak** [haijeploeg/excludarr#62]
**Describe the bug**
I want to keep the shows on sonarr as unmonitored but with files deleted

**To Reproduce**
I added sopronos from hbo and the expanse from amazon prime with pilot episode monitored to test.

i ran the command docker run -it --rm --env-file excludarr.env haijeploeg/excludarr:latest sonarr exclude -a not-monitored -d -y

**Expected behavior**
i may have mis-understood, but I thought this would unmonitor the show, but they still show as monitored.
**Versions**
1. OS Version: ... docker imagee
2. Excludarr version: ...
3. Python version: ...

**Debug logging**
To debug the problem further, we need to know exactly what went wrong. Therefor we ask you to run the command in debug mode. You can achieve this by appending the `--debug` flag after the base command `excludarr`. e.g. `excludarr --debug sonarr exclude -a delete -d -e`.

If the output is too long, please paste the output to pastebin or github gist and link it in this issue.

**Additional context**

**Comment by haijeploeg** [comment]
I cannot answer this as the template is not filled in properly. I need the output of the `--debug` flag and I need your configuration (you can strip the api keys). Without that information I cannot answer and/or debug this properly.

**Comment by Vansmak** [comment]
> I cannot answer this as the template is not filled in properly. I need the output of the `--debug` flag and I need your configuration (you can strip the api keys). Without that information I cannot answer and/or debug this properly.

https://pastebin.com/Y1VPQZjG debug

env 
GENERAL_FAST_SEARCH=true
GENERAL_LOCALE=en_US
GENERAL_PROVIDERS=hbo, hbo max, amazon prime video
RADARR_URL=http://192.168.254.210:7878
RADARR_API_KEY=**************
RADARR_VERIFY_SSL=false
SONARR_URL=http://192.168.254.210:8989
SONARR_API_KEY=**************
SONARR_VERIFY_SSL=false

**Comment by Vansmak** [comment]
No actual files\episodes but stays monitored

**Comment by Vansmak** [comment]
Also as a new feature it would be nice if it coukd add a tag to shows or movies like the app elsewherr does.  Then one coukd use profiles in sonarr to control if monitored or downloaded.  If ant show us available i  one of the listed providers tag it with some word.   -t tagword

**Comment by haijeploeg** [comment]
Could you post a screenshot of the problem? It should delete the files and unmonitor the specific episodes/seasons. It will only disable monitoring of the whole show if the show has ended. Sometimes a new season will not end up on any of the providers, therefor you want to monitor the upcoming seasons.

The tag feature is already a feature request and will be implemented soon.

**Comment by Vansmak** [comment]
Using this as an example the show is still monitored after i ran excludarr. 

So glad to hear tagging is on the way i was about to start hacking code and i am not a coder.

**Comment by haijeploeg** [comment]
Yes that is expected behaviour. If you look at the seasons and the episodes, none of them are monitored anymore. The show itself is still monitored in case a new season is being released. It could be that the season being released is not on any streaming provider, and therefor excludarr is keeping the show listed as monitored. 

Practically all the seasons and episodes are being deleted and not being monitored anymore, only the future seasons, not listed in Sonarr, will be monitored after public confirmation.

When a show ends, so when a show is really done and it has been confirmed that no seasons are being released anymore, then the show will disable monitoring.

**Comment by Vansmak** [comment]
Ok ty for explaining, kinda surprised Sopranos is also showing as monitored and it ended in 2007.  No big deal because there will be no new episodes, but for shows that will have a new season but are in one of my streaming platforms id like it to not monitor or dl anything for that show.  Just leave it as unmonitored but dont remove it.

Ill work around it. Ty again.
